discoveries/projected-changes-WUS-snow.discoveries.mdx

```diff
@@ -0,0 +1,142 @@
+
+  Join the discussion and provide comments on this Discovery at https://github.com/orgs/Earth-Information-System/discussions.
+
+  Authors: Justin Pflug, Sujay Kumar, Ben Livneh, Melissa Wrzesien, Kim Locke
+  </Prose>
+</Block>
+
+<Block>
+  <Prose>
+  ## Climate change projections
+   Climate projections came from the [Climate Model Intercomparison Project, Phase 6](https://www.wcrp-climate.org/wgcm-cmip/wgcm-cmip6). CMIP6 Projections of air temperature, precipitation, relative humidity, wind speed, and radiation were downscaled to finer spatial resolutions by the NASA Earth Exchange Global Daily Downscaled Projections ([NEX-GDDP](https://www.nccs.nasa.gov/services/data-collections/land-based-products/nex-gddp-cmip6)). Projections show that most of the Western US average winter air temperature and precipitation are expected to increase by the end of the century (2076 - 2095), relative to the historical timeframe (1995 - 2016).
+  </Prose>
+  <Figure>
+    <Map
+      datasetId='CMIP-winter-median-ta'
+      layerId='CMIP245-winter-median-ta'
+      dateTime='2085-02-14'
+      zoom={4.0}
+      center={[-113.,40.]}
+      compareDateTime='2099-01-01'
+      compareLabel='Slide to see the underlying terrain'
+    />
+    <Caption
+      attrAuthor='NASA'
+      attrUrl='https://nasa.gov/'
+    >
+    Median projected changes to end-of-century winter air temperature from a 23-member ensemble of CMIP6 models (SSP2-4.5).
+</Caption>
+  </Figure>
+</Block>
+<Block>
+  <Figure>
+    <Map
+      datasetId='CMIP-winter-median-pr'
+      layerId='CMIP245-winter-median-pr'
+      dateTime='2085-02-14'
+      zoom={4.0}
+      center={[-113.,40.]}
+      compareDateTime='2099-01-01'
+      compareLabel='Slide to see the underlying terrain'
+    />
+    <Caption
+      attrAuthor='NASA'
+      attrUrl='https://nasa.gov/'
+    >
+    Median projected percent-changes to end-of-century total winter precipitation from a 23-member ensemble of CMIP6 models (SSP2-4.5).
+</Caption>  
+  </Figure>
+</Block>
+
+<Block type='wide'>
```

Comment:
Consider whether to keep the `wide` block for such little text, as it shows up in a partial column looking broken.

<img width="1217" alt="image" src="https://user-images.githubusercontent.com/1090606/231118897-199aac9e-52ad-41c1-b987-64ce44d61b0a.png">